pyproject.toml

```diff
@@ -38,7 +38,7 @@ dependencies = [
     "natsort>=7.1.0",
     "networkx>=2.6, <=3.3",
     "ninja>=1.10.0.post2, <1.12",
-    "numpy>=1.19.1, <1.27",
+    "numpy>=1.19.1, <2.2.0",
```

Comment:
```suggestion
    "numpy>=1.19.1",
```

Reply:
Why?

Reply:
Do you think that it can be done. Honestly, I see only one reason for the upper limit, so that we can constantly move the version.